upup/pkg/fi/cloudup/scaleway/OWNERS

```diff
@@ -0,0 +1,3 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+labels:
+- area/provider/scaleway
```

Comment:
Same comment:
Add a new line at the end.
Let's also designate the ownership of Scaleway to
https://github.com/orgs/scaleway/teams/containers
and https://github.com/orgs/scaleway/teams/devtools